### PR TITLE
Update storage-aws.md

### PR DESCRIPTION
**Pull request by amckinley**
Improve clarity around blocks vs chunks storage options.

### CODE REVIEW

File: docs/production/storage-aws.md [hunk 1]
```diff
@@ -18,8 +18,10 @@ if you use MFA), or use a short-term token solution such as
 
 ## Should I use S3 or DynamoDB ?
 
-Note that the choices are: "chunks" of timeseries data in S3 and index
-in DynamoDB, or everything in DynamoDB. Using just S3 is not an option.
+Note that the choices for the chunks stoage backend are: "chunks" of 
+timeseries data in S3 and index in DynamoDB, or everything in DynamoDB. 
+Using just S3 is not an option, unless you use the experimental "blocks"
+storage engine.
```
Comment (pracucci) [inline docs/production/storage-aws.md:24]:
```suggestion
Using just S3 is not an option, unless you use the experimental [blocks storage engine](../blocks-storage/).
```

**Comment by pstibrany** [comment]
@amckinley Would you mind fixing DCO please, so that we can merge this?

**Review by pstibrany — APPROVED** [review]
LGTM, with Marco's suggestion about the link.

File: docs/production/storage-aws.md [hunk 2]
```diff
@@ -18,8 +18,10 @@ if you use MFA), or use a short-term token solution such as
 
 ## Should I use S3 or DynamoDB ?
 
-Note that the choices are: "chunks" of timeseries data in S3 and index
-in DynamoDB, or everything in DynamoDB. Using just S3 is not an option.
+Note that the choices for the chunks stoage backend are: "chunks" of
```
Comment (forestsword) [inline docs/production/storage-aws.md:21]:
```suggestion
Note that the choices for the chunks storage backend are: "chunks" of 
```

**Comment by pracucci** [comment]
_I'm taking over this PR to get it merged._

**Pull request merged**